pages/common/mpc.md

```diff
@@ -0,0 +1,28 @@
+# mpc
+
+> Music Player Client.
+> Program for controlling Music Player Daemon (MPD).
```

Comment:
This sentence construction looks a bit awkward to me. I understand why it's like this, but wouldn't "the Music Player Daemon" sound more natural? Just asking, I could be persuaded otherwise.